Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -326,6 +344,8 @@ def generate_search_query(time_to_fetch_from: Optional[datetime],
         messages_query = OR(from_=permitted_from_addresses_list + permitted_from_domains_list).format()
         # Removing Parenthesis and quotes
         messages_query = messages_query.strip('()').replace('"', '')
+        if mail_service == 'Office365':
+            messages_query = messages_query.replace('FROM', 'HEADER FROM')
```

Comment:
i dont really like this solution, adding an integration param seems like an overkill

can we query with both `FROM` and `HEADER FROM` ?

Reply:
In the mail service we checked with (Gmail) it is possible but to keep BC (and to ensure work with all mail services) we preferred to add a parameter
@reutshal FYI